website/content/docs/v0.1/Configuration/serverclasses.md

```diff
@@ -5,29 +5,60 @@ weight: 3
 
 # Server Classes
 
-Server classes are a way to group distinct server resources.
-The "qualifiers" key allows the administrator to specify criteria upon which to group these servers.
-There are currently three keys: `cpu`, `systemInformation`, and `labelSelectors`.
-Each of these keys accepts a list of entries.
-The top level keys are a "logical AND", while the lists under each key are a "logical OR".
-Qualifiers that are not specified are not evaluated.
+Server classes are a way to group distinct server resources.  The `qualifiers`
```

Comment:
We run linters that should be enforcing https://sembr.org. Not sure why this isn't caught, but fair game if it is broken. Just FYI. TL;DR is that there should be a sentence per-line. I am not going to block on it since our linting isn't catching it.

Reply:
https://github.com/talos-systems/sidero/pull/422

Reply:
That PR is merged now, linting should catch that

Reply:
Fixed the style. Thanks.